File: app/src/main/java/dev/ragnarok/fenrir/util/Analytics.java
package dev.ragnarok.fenrir.util;

import dev.ragnarok.fenrir.Constants;


public class Analytics {

    public static void logUnexpectedError(Throwable throwable) {
        if (Constants.IS_DEBUG) {
            throwable.printStackTrace();
        }

        //FirebaseCrash.report(throwable);
    }
}

File: app/src/main/java/dev/ragnarok/fenrir/util/Analytics.kt
package dev.ragnarok.fenrir.util

import dev.ragnarok.fenrir.Constants

object Analytics {

    fun logUnexpectedError(throwable: Throwable?) {
        if (Constants.IS_DEBUG) {
            throwable?.printStackTrace()
        }

        //FirebaseCrash.report(throwable)
    }
}